src/components/repository-profile.component.js

```diff
@@ -141,6 +143,10 @@ export const RepositoryProfile = ({
         {emojifyText(repository.description) || ' '}
       </Text>
 
+      <Text style={styles.languageInfoTitle}>
+        {subscribed ? '(watching)' : undefined}
```

Comment:
It needs to be moved to the local file (`src/locale/languages/en.js`) and output via [translate](https://github.com/gitpoint/git-point/blob/master/src/utils/method-helpers.js#L33) helper.

And yet, can it be better to return `null` instead of `undefined`?